proto/regen/ecocredit/v1beta1/state.proto

```diff
@@ -12,13 +11,13 @@ message ClassInfo {
 
     option (cosmos.orm.v1alpha1.table) = {
         id: 1,
-        primary_key: {fields: "class_id"}
+        primary_key: {fields: "id", auto_increment: true}
         index: {id: 1 fields: "admin"}
         index: {id: 2 fields: "credit_type"}
     };
 
-    // class_id is the unique ID of credit class.
-    string class_id = 1;
+    // id is the unique ID of credit class.
+    uint64 id = 1;
```

Comment:
The previous class id is being used to generate batch denoms. Do we need an `abbreviation` field to retain that piece of information? Seems like we can just handle formatting when creating a batch.

Reply:
That being said, are we still planning to track the sequence for each credit type and use `C01`, `C02`, `B01`, etc. or will `B01` in this case become `B03`?

Reply:
Yeah I'm not sure if we need to store that or not here, but we can.

Sequences for credit types wasn't the intended behavior. We just want a simple `C` or `B` that's chosen by governance to be unique (1-3 characters).

Reply:
So I added back a class seq scoped to credit type, but renamed the table to `ClassSequence`. It was confusing to call it CreditTypeSeq, because the seq is for the class within the type, not for the credit type.

So classes will still be `C01`, etc.